.env-sample

```diff
@@ -14,3 +18,5 @@ MAIL_TRAP_HOST=<host>
 MAIL_TRAP_PORT=<port>
 MAIL_TRAP_USER=<userID>
 MAIL_TRAP_PASS=<userPassword>
+
+
```

Comment:
You want to always have only one newline at the end of your files

Reply:
ok